### PR TITLE
Fix styles-ssr timeout

### PR DESCRIPTION
**Pull request by drwpow**
## Changes

Minor fix: removes unnecessary timeout setting for test

## Testing

Tests should pass

## Docs

No docs needed

### CODE REVIEW

File: packages/astro/test/astro-styles-ssr.test.js [hunk 1]
```diff
@@ -2,27 +2,25 @@ import { expect } from 'chai';
 import cheerio from 'cheerio';
 import { loadFixture } from './test-utils.js';
 
-describe('Styles SSR', function () {
-  this.timeout(10000);
```
Comment (drwpow) [inline packages/astro/test/astro-styles-ssr.test.js:6]: Not necessary

Comment (jonathantneal) [reply packages/astro/test/astro-styles-ssr.test.js:6]: ✨

**Review by jonathantneal — APPROVED** [review]
So these new tests fix issues and are clearer to read. Nice!

**Comment by matthewp** [comment]
I added that timeout because it runs a little slower in the OSX CI and it actually timed out a few times in PRs I worked on.  We can monitor it though and add back later if needed.

Also, the reason for putting the `before` blocks inside of the describe is so that if the before fails mocha can tell you which test suite is failing. When they are at the top-level and you run `yarn test` you will see a test is failing, but not know what file it is coming from because Mocha doesn't have the context to tell you.

**Comment by drwpow** [comment]
> I added that timeout because it runs a little slower in the OSX CI and it actually timed out a few times in PRs I worked on. We can monitor it though and add back later if needed.

Oh yeah I was seeing the same—I ran out of time on those tests. But we bumped up the default timeout to `15s` in `package.json` so this was now slower than that.

> Also, the reason for putting the `before` blocks inside of the describe is so that if the before fails mocha can tell you which test suite is failing. When they are at the top-level and you run `yarn test` you will see a test is failing, but not know what file it is coming from because Mocha doesn't have the context to tell you.

Oh! I’m sorry. Good to know. That may be a good idea to just do that for everything from now on.

**Pull request merged**